src/tests/registration/Registration.int.test.tsx

```diff
@@ -0,0 +1,261 @@
+      {
+        target: { value: TestValue.Email },
+      }
+    );
+    fireEvent.change(
+      getByTestId(`${StudentRole.PARENT} ${DefaultFields.ADDRESS.name}`),
+      {
+        target: { value: TestValue.Address },
+      }
+    );
+    fireEvent.change(
+      getByTestId(`${StudentRole.PARENT} ${TEST_DYNAMIC_FIELD.name}`),
+      {
+        target: { value: TestValue.ParentFavouriteColour },
+      }
+    );
+
+    // children section
+    fireEvent.change(
+      getByTestId(`${StudentRole.CHILD} ${DefaultFields.FIRST_NAME.name}`),
+      {
+        target: { value: TestValue.ChildFirstName },
+      }
+    );
+    fireEvent.change(
+      getByTestId(`${StudentRole.CHILD} ${DefaultFields.LAST_NAME.name}`),
+      {
+        target: { value: TestValue.LastName },
+      }
+    );
+
+    const childDobInput = getByLabelText(
+      `${StudentRole.CHILD} ${DefaultFields.DATE_OF_BIRTH.name}`
+    ) as HTMLInputElement;
+    childDobInput.setSelectionRange(0, childDobInput.value.length);
+    userEvent.type(childDobInput, TestValue.ChildDob);
+
+    fireEvent.change(
+      getByTestId(`${StudentRole.CHILD} ${TEST_DYNAMIC_FIELD.name}`),
+      {
+        target: { value: TestValue.ChildFavouriteColour },
+      }
+    );
+
+    // guest section
+    fireEvent.click(getByRole("button", { name: "Add member" }));
+    fireEvent.change(
+      getByTestId(`${StudentRole.GUEST} ${DefaultFields.FIRST_NAME.name}`),
+      {
+        target: { value: TestValue.GuestFirstName },
+      }
+    );
+    fireEvent.change(
+      getByTestId(`${StudentRole.GUEST} ${DefaultFields.LAST_NAME.name}`),
+      {
+        target: { value: TestValue.LastName },
+      }
+    );
+
+    // session section
+    fireEvent.change(
+      getByTestId(`${StudentRole.PARENT} ${TEST_SESSION_DYNAMIC_FIELD.name}`),
+      {
+        target: { value: TestValue.TimeInCanada },
+      }
+    );
+
+    const guestDobInput = getByLabelText(
+      `${StudentRole.GUEST} ${DefaultFields.DATE_OF_BIRTH.name}`
+    ) as HTMLInputElement;
+    guestDobInput.setSelectionRange(0, guestDobInput.value.length);
+    userEvent.type(guestDobInput, TestValue.GuestDob);
+
+    fireEvent.change(
+      getByTestId(`${StudentRole.GUEST} ${TEST_DYNAMIC_FIELD.name}`),
+      {
+        target: { value: TestValue.GuestFavouriteColour },
+      }
+    );
+
+    fireEvent.change(getByTestId(TestId.PreferredClassSelect), {
+      target: { value: session.classes[1].id },
+    });
+
+    fireEvent.change(getByTestId(TestId.StatusSelect), {
+      target: { value: EnrolmentStatus.SIGNED_UP },
+    });
+
+    fireEvent.change(getByTestId(TestId.NotesInput), {
+      target: { value: TestValue.Notes },
+    });
+
+    fireEvent.click(getByRole("button", { name: "Done" }));
+
+    await waitFor(() =>
+      expect(EnrolmentAPI.postEnrolment).toHaveBeenCalledTimes(1)
+    );
+    expect(EnrolmentAPI.postEnrolment).toHaveBeenCalledWith({
+      family: {
```

Comment:
when https://github.com/uwblueprint/project-read-backend/pull/110 goes in, we'll be expecting to post with a family ID instead right?